### DISCUSSION TITLE
2.0.0-beta.16 problem

### DISCUSSION THREAD

**Discussion by JohnTelford** [facebook/docusaurus#6770] [Q&A]
I was testing 2.0.0-beta.16 on Cloudflare. It fails and reports:
error @docusaurus/core@2.0.0-beta.16: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.18.0"
--

Previously I deployed the same code to Cloudflare with 2.0.0-beta.15 . It did not have any problems.

My question is how do I reinstall 2.0.0-beta.15 so I can continue my work?

Thanks... John Telford

**Comment by Josh-Cena** [comment]
Hi, you should upgrade your Cloudflare's Node version to 14+. There's no reason to continue working on Node 12.
